client/my-sites/site-settings/comment-display-settings/index.jsx

```diff
@@ -0,0 +1,85 @@
+/**
+* External dependencies
+*/
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import FormSelect from 'components/forms/form-select';
+import FormTextInput from 'components/forms/form-text-input';
+import JetpackModuleToggle from '../jetpack-module-toggle';
+import { isJetpackModuleActive } from 'state/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+
+class CommentDisplaySettings extends Component {
+	shouldEnableSettings() {
+		const { isCommentsModuleActive, submittingForm } = this.props;
+		return !! submittingForm || ! isCommentsModuleActive;
+	}
+
+	render() {
+		const {
+			fields,
+			onChangeField,
+			selectedSiteId,
+			submittingForm,
+			translate,
+		} = this.props;
+
+		return (
+			<FormFieldset className="comment-display-settings">
+				<JetpackModuleToggle
+					siteId={ selectedSiteId }
+					moduleSlug="comments"
+					label={ translate(
+						'Use Jetpack Comments. Readers will be able to use their WordPress.com, Twitter, Facebook, or Google+ accounts ' +
+						'to leave comments.'
+					) }
+					disabled={ !! submittingForm }
+					/>
+				<div className="comment-display-settings__module-setting is-indented">
+					<FormLabel htmlFor="highlander_comment_form_prompt">{ translate( 'Comments Label' ) }</FormLabel>
```

Comment:
Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'Comment and Featured Labels' )` *ES Score: 7.78*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).